CHANGELOG.md

```diff
@@ -6,6 +6,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [1.1.0]
+### Added
+- add build command option ([#11](https://github.com/MetaMask/action-publish-gh-pages/pull/11))
+### Removed
+- removed `setup` from being called
```

Comment:
Nit: I noticed one was imperative and one was past-tense, so I've made them both imperative. Also now they're capitalized, and both reference the PR the change was in.
```suggestion
- Add build command option ([#11](https://github.com/MetaMask/action-publish-gh-pages/pull/11))
### Removed
- Remove `setup` npm script call ([#11](https://github.com/MetaMask/action-publish-gh-pages/pull/11))
```